docs/theme/js/theme.js

```diff
@@ -1,10 +1,188 @@
+
+(function ($) {
```

Comment:
Since `theme.js` looks identical to teleport docs, it seems that it makes sense to move this code to `web` repo and expose it via grvLib.